lib/src/sv_common.c

```diff
@@ -1027,6 +1044,9 @@ signed_video_free(signed_video_t *self)
 
   // Free the legacy validation if present.
   legacy_sv_free(self->legacy_sv);
+  if (self->onvif) {
```

Comment:
No need to check if `self->onvif` exists. The free function can handle a null pointer.